### PR TITLE
TRIVIAL: Update changelog

### PR DESCRIPTION
**Pull request by strouhaljan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,14 @@ The REST API versions in the table are just for your information as the values a
 
 - We ask developers to consider using the Headline component instead of the KPI component. The KPI component may be eventually marked as deprecated in one of the next major versions.
 
+## 6.3.1
+
+April 26, 2019
+
+### Changed
+
+- Use updated @gooddata/goodstrap library to fix error when installing dependencies on MS Windows machines ([commit](https://github.com/gooddata/gooddata-react-components/pull/929/commits/a6e4ab0acf91bb4318d8ffb2c394beae14adb125))
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md]: Use the updated @gooddata/goodstrap library to fix the error that occurs when installing dependencies on MS Windows machines

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3336/ : SUCCESS in 4m 27s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4133/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build in pipeline gate aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1787/ : SUCCESS in 21m 51s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3350/ : FAILURE in 17m 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/917/ : SUCCESS in 29s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/969/ : SUCCESS in 30m 24s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1790/ : SUCCESS in 29m 50s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3354/ : SUCCESS in 3m 01s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/919/ : SUCCESS in 29s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/972/ : SUCCESS in 12m 08s

**Pull request merged**